sdk/metric/example_test.go

```diff
@@ -58,3 +61,134 @@ func Example() {
 	// See the go.opentelemetry.io/otel/metric package for more information
 	// about the metric API.
 }
+
+func ExampleView() {
```

Comment:
Those examples create a view, but never assign it to a meter provider. Should we cover it too?

Reply:
I would prefer to address it in a separate PR (baby steps). Here I only moved the existing examples. However, I think we can keep the discussion.

> Should we cover it too?

I think that yes 😉 At least in the main `ExampleView`.

Reply:
@dmathieu Maybe that will be good enough for now d063d777?